README.md

```diff
@@ -6,17 +6,19 @@ This is the Python examples repository for [LUSID by FINBOURNE](https://www.finb
 
 ## Contents
 
-The demo scripts show the SDK in a natural setting and show how the Lusid SDK can be used for a variety of operations.
+The demo scripts show the SDK in a natural setting and show how the LUSID SDK can be used for a variety of operations.
```

Comment:
can we make this consistent with the C# (https://github.com/finbourne/lusid-sdk-examples-csharp/pull/22/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R8) I think that reads better